jwala-services/src/main/java/com/cerner/jwala/service/app/impl/ApplicationServiceImpl.java

```diff
@@ -220,6 +220,7 @@ public CommandOutput deployConf(final String appName, final String groupName, fi
                         "The target JVM must be stopped before attempting to update the resource files");
             }
             final String hostName = jvm.getHostName();
+            historyFacadeService.write("", jvm.getGroups(), "Generating and Deploying \"" + appName + "\" resources", EventType.USER_ACTION_INFO, user.getId());
```

Comment:
Can we get a more detailed message than just a generic one? Can we get one log entry for each webapp resource that we generate?